src/modules/uuv_att_control/uuv_att_control.cpp

```diff
@@ -106,12 +106,23 @@ void UUVAttitudeControl::vehicle_attitude_setpoint_poll()
 {
 	bool updated = false;
 	orb_check(_vehicle_attitude_sp_sub, &updated);
+	warn("Setpoint: %d", updated);
```

Comment:
Needed? This could be quite noisy.

Reply:
Oh, this line got there by accident! I removed it in 7f2762746e5eb64800a10a78e65772fa1d2f71ec